.github/workflows/ci-actions-incremental.yml

```diff
@@ -126,12 +126,12 @@ jobs:
           GIB_ARGS="-Dincremental -Dgib.disableSelectedProjectsHandling -Dgib.untracked=false -Dgib.uncommitted=false"
           if [ -n "$PULL_REQUEST_BASE" ]
           then
-              # The PR defines a clear merge target so just use that branch for reference, *unless*
+              # The PR defines a clear merge target so just use that branch for reference, *unless*:
               # - the current branch is a backport branch
-              GIB_ARGS+=" -Dgib.referenceBranch=$PULL_REQUEST_BASE -Dgib.disableIfBranchRegex='.*backport.*'"
+              GIB_ARGS+=" -Dgib.referenceBranch=origin/$PULL_REQUEST_BASE -Dgib.disableIfBranchRegex='.*backport.*'"
```

Comment:
Output of the checkout action for this PR:
```
   Initialized empty Git repository in /home/runner/work/quarkus/quarkus/.git/
  /usr/bin/git remote add origin https://github.com/quarkusio/quarkus
```
which is correct (not the fork).

Output from PRs in my fork (which nobody will do, but just to illustrate it does the right thing):
```
   Initialized empty Git repository in /home/runner/work/quarkus/quarkus/.git/
  /usr/bin/git remote add origin https://github.com/famod/quarkus
```